hack/dockerized

```diff
@@ -14,7 +14,7 @@ SYNC_VENDOR=${SYNC_VENDOR:-false}
 TEMPFILE=".rsynctemp"
 
 # Build the build container
-(cd ${DOCKER_DIR} && docker build . -q -t ${BUILDER})
+(cd ${DOCKER_DIR} && docker build . -t ${BUILDER})
```

Comment:
Hm, without the `-q` starting `hack/dockerized` is very verbose.

Reply:
Well, with the -q it is completely silent, sometimes for several minutes.  My preference is to always have some feedback, but if you insist, I can put it back.

Reply:
Ok, so lets do the following: If "JOB_NAME" (jenkins) or "TRAVIS_BUILD_ID" (travis) is defined, then add the `-q`. Otherwise not.

Only parts of the testing infra cache the builder and it is normally only of interest when something goes wrong during creating the builder, where you then get reasonable output. For devs it is either cached (so almost no extra output), or the situation you describe is encountered, where you did not see output for minutes. There it is probably always preferred to see the progress.

What do you think?

An alternative would also be to simply print a message like `Creating the build container. The initial uncached build may take several minutes.`